pkg/scalers/azure_queue.go

```diff
@@ -18,11 +18,7 @@ func GetAzureQueueLength(ctx context.Context, usePodIdentity bool, connectionStr
 	if !usePodIdentity {
 
 		var accountKey string
-<<<<<<< HEAD
```

Comment:
This looks like a bad merge :D

Reply:
Yes indeed! Rookie mistake :) Fixed now!

Reply:
No worries!